### PR TITLE
Update to rollups 0.9.1 and use sunodo

### PR DESCRIPTION
**Pull request by miltonjonat**
**Major changes**
- Eliminating docker boilerplate in favor of sunodo
- ABIInputCodec now can specify packed or non-packed encoding, and the header is optional
- Router includes the option to route by msg_sender

**Things to improve**
- Wallet spec not fully implemented: should emit notices for deposits and withdrawals, and emit reports in case of error
- Typings: wagmi specifies types as `0x${string}`, maybe we should use that throughout deroll to make things easier
- Deployments address book: this PR currently implements a very stupid AddressBook module with constants. Maybe we should get deployments from the npm, or at least make the module's API have methods so that dynamic addresses per network are possible
- Improve sunodo build to have a proper build stage and avoid having to run `yarn build` outside first. For that to work, the context must be changed to the root directory

### CODE REVIEW

File: apps/backend/package.json [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
     "name": "ballaum-backend",
-    "version": "0.1.0",
+    "version": "0.2.0",
     "description": "Ballaum DApp",
     "main": "dist/index.js",
     "author": "Danilo Tuler",
     "license": "MIT",
     "scripts": {
-        "build": "ts-node esbuild.ts",
+        "build": "ts-node esbuild.ts --bundle --outfile=dist/index.js --platform=node --target=node19",
```
Comment (tuler) [inline apps/backend/package.json:9]: why override here? if everything is in `esbuild.ts`

**Review by tuler — COMMENTED** [review]
Looking good so far

File: apps/backend/package.json (hunk 1 above)
Comment (miltonjonat) [reply apps/backend/package.json:9]: Ignorance of mine.

**Comment by tuler** [comment]
Supersede by #2

**Pull request closed without merging**